pkg/kubelet/dockertools/manager.go

```diff
@@ -1494,14 +1494,28 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	if err != nil {
 		return kubecontainer.ContainerID{}, fmt.Errorf("InspectContainer: %v", err)
 	}
+
+	if !containerInfo.State.Running {
```

Comment:
I'd prefer the latter. Skip applying oom-score adjustment if the pid is `0` or `1`.